controllers/request.js

```diff
@@ -0,0 +1,80 @@
+//Request controller
+
+const Requests = require('../models/Requests');
+const Users = require('../models/Users');
+const Teams = require('../models/Teams');
+
+const deleteRequest = async (req, res) =>{
+    Request.deleteOne({_id:req.body.requestId});
+}
+
+const sendRequest = async (req, res) =>{    
+    const user = await Users.find({_id: req.body.user_id});
+    const team = await Teams.find({_id: req.body.team_id});
+
+    if (team.Member_count < 3){
+        //Save a new request to the database
+        const request = new Requests({
+            For_team: req.body.team_id,
+            Req_to: req.body.user_email
+        });
+        request.save();
+    
+        if (!team || !user){
+            res.status(404).send('Team or user not found.');
+        }
+        
+        //Add request to team
+        
+        team.Pending_Request = team.Pending_Request.concat([{
+            type: request._id
+        }]);
+
+        //Add request to user
+
+        user.Pending_Request = user.Pending_Request.concat([{
+            type: request._id
+        }]);
+    }
+};
+
+const acceptRequest = async (req, res) =>{
+    const team = await Teams.find({_id: req.body.team_id});
+    const user = await Users.find({_id: req.body.user_id});
+
+    if (team.Member_count >= 3){
```

Comment:
The condition here should be Member_count >=2 because the the number of members would get incremented by 1 upon accepting the current request